tauri-app/src/lib/utils/number.ts

```diff
@@ -0,0 +1,3 @@
+export function isStringValidNumber(val: string) {
+  return /^\d+.?[\d]+$/.test(val);
```

Comment:
can we get some test cases around this?

Reply:
removed and replaced with [@imask/svelte](https://www.npmjs.com/package/@imask/svelte)